era1.tf

```diff
@@ -0,0 +1,9 @@
+/* ERA1 Files hosting */
+
+resource "cloudflare_record" "era1_sepolia" {
```

Comment:
Shouldn't this be part of `sepolia.tf` ? to have all sepolia terraform resource in the same place ?

Reply:
ah sure yeah I can move it there.

Reply:
done.